static/js/src/public/details/index.js

```diff
@@ -31,8 +31,37 @@ const truncateSummary = (selector) => {
         summaryContentEl.innerHTML = summary;
       });
     }
-  } else {
-    throw new Error(`There are no elements containing ${selector} selector.`);
+  }
+};
+
+const handleTopology = () => {
+  const topologyInfoPanel = document.querySelector("[data-js='topology-info']");
+  const topologyModal = document.querySelector("[data-js='topology-modal']");
+
+  if (topologyInfoPanel && topologyModal) {
+    const de = document.documentElement;
+    const vw = Math.max(de.clientWidth, window.innerWidth || 0);
+    const vh = Math.max(de.clientHeight, window.innerHeight || 0);
+    const delta = 300;
+    const width = vw - delta;
+    const height = vh - delta;
+    const closeModalButton = topologyModal.querySelector(
+      "[aria-controls='modal']"
+    );
+    const topologyImage = topologyModal.querySelector("object");
+
+    topologyImage.style.width = `${width}px`;
+    topologyImage.style.height = `${height}px`;
```

Comment:
Why not use CSS to style and position the topology object? You will need to handle resizes if you are styling with us.

Reply:
Agree. thanks